src/backends/torch/torchlib.cc

```diff
@@ -1033,7 +1073,7 @@ namespace dd
         throw;
       }
     this->_stats.transform_end();
-
+    _module.to(_dtype);
```

Comment:
Can we do this only if the type has changed ? Or this call has no direct performance impact ?

Reply:
check is done internally in torch (according to pytorch doc)